app/services/builders/external_project.rb

```diff
@@ -6,9 +6,10 @@ def initialize(repository_data)
 
     def call
       ::ExternalProject.find_or_initialize_by(github_id: @repository_data[:id]) do |project|
-        project.name = @repository_data[:name]
-        project.full_name = @repository_data[:full_name]
-        project.description = @repository_data[:description]
+        name = @repository_data[:name]
+
+        project.name = name.split('/').second
+        project.full_name = name
       end
```

Comment:
We could now save the project here, right? Because if we call this object we know for sure we want to keep the project. (And we could get rid of `Builders::ExternalPullRequest#built_or_saved_project`)